docs/proposals/lifelong-learning.md

```diff
@@ -0,0 +1,185 @@
+* [Lifelong Learning](#lifelong-learning)
+   * [Motivation](#motivation)
+     * [Goals](#goals)
+     * [Non\-goals](#non-goals)
+   * [Proposal](#proposal)
+     * [Use Cases](#use-cases)
+   * [Design Details](#design-details)
+     * [CRD API Group and Version](#crd-api-group-and-version)
+     * [Lifelong learning CRD](#lifelong-learning-crd)
+     * [Lifelong learning type definition](#lifelong-learning-job-type-definition)
+     * [Lifelong learning sample](#lifelong-learning-job-sample)
+     * [Validation](#validation)
+   * [Controller Design](#controller-design)
+     * [Lifelong Learning Controller](#lifelong-learning-controller)
+     * [Downstream Controller](#downstream-controller)
+     * [Upstream Controller](#upstream-controller)
+     * [Details of api between GM(cloud) and LC(edge)](#details-of-api-between-gmcloud-and-lcedge)
+   * [Workers Communication](#workers-communication)
+
+# Lifelong Learning
+## Motivation
+
+
+The edge-cloud collaborative lifelong learning combines the attributes of multi-task learning and incremental learning at the edge to deal with the problem of heterogeneous data and small samples in complex scenarios. Rely on the memory of the knowledge base to adapt to new situations, thus effectively solving the small sample problem of traditional machine learning.
```

Comment:
Motivation is important and thus careful polishing would be needed. My suggestion is to modify the writing a little bit like "At present, edge-cloud synergy machine learning is confronted with the challenge of heterogeneous data distributions in complex scenarios and small samples on the edge. The edge-cloud synergy lifelong learning is accordingly proposed: 1) In order to learn with shared knowledge between historical scenarios, the scheme is essentially the combination of another two learning schemes, i.e., multi-task learning and incremental learning; 2) The cloud knowledge base in lifelong learning empowers the scheme with memory ability, which helps to adapt historical knowledge to new and unseen situations on the edge. Joining the forces of multi-task learning, incremental learning and the knowledge base, the lifelong learning scheme seeks to fundamentally overcome the above challenges of edge-cloud synergy machine learning."

Reply:
fix